./docker/script/dev_start.sh  apollo_2    提示：Warning: Unknown option: apollo_2

idriver@idriver-System:~/apollo-master-2$ **./docker/scripts/dev_start.sh apollo_2**
Warning: Unknown option: apollo_2

idriver@idriver-System:~/apollo-master-2$ **./docker/scripts/dev_start.sh **-image** apollo_2**
[INFO] Start pulling docker image apolloauto/apollo:apollo_2 ...
Error response from daemon: manifest for apolloauto/apollo:apollo_2 not found
[ERROR] Failed to pull docker image.

我确定已经执行了docker commit xxxxx apolloauto/apollo：apollo_2，但是无法pull
请教各位大神，非常感谢！

--- a/docker/scripts/dev_start.sh
+++ b/docker/scripts/dev_start.sh
@@ -17,6 +17,7 @@
 ###############################################################################
 
 INCHINA="no"
+LOCAL_IMAGE="no"
 VERSION=""
 ARCH=$(uname -m)
 VERSION_X86_64="dev-x86_64-20180103_1300"
@@ -31,6 +32,7 @@ OPTIONS:
     -C  pull docker image from China mirror
     -h, --help   display this help and exit
     -image <version>    specify which version of a docker image to pull
+    -l   use local docker image
 EOF
 exit 0
 }
@@ -57,6 +59,9 @@ do
         ;;
     -h|--help)
         show_usage
+        ;;
+    -l|--local)
+        LOCAL_IMAGE="yes"
         ;;
     *)
         echo -e "\033[93mWarning\033[0m: Unknown option: $1"
@@ -98,11 +103,15 @@ source ${APOLLO_ROOT_DIR}/scripts/apollo_base.sh
 
 function main(){
 
-    info "Start pulling docker image $IMG ..."
-    docker pull $IMG
-    if [ $? -ne 0 ];then
-        error "Failed to pull docker image."
-        exit 1
+    if [ "$LOCAL_IMAGE" = "yes" ];then
+        info "Start docker container based on local image : $IMG"
+    else
+        info "Start pulling docker image $IMG ..."
+        docker pull $IMG
+        if [ $? -ne 0 ];then
+            error "Failed to pull docker image."
+            exit 1
+        fi
     fi
 
     docker ps -a --format "{{.Names}}" | grep 'apollo_dev' 1>/dev/null

